build.sbt

```diff
@@ -0,0 +1,74 @@
+import sbt.Tests.{Group, SubProcess}
+import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin._
+import uk.gov.hmrc.SbtAutoBuildPlugin
+import scalariform.formatter.preferences._
+
+lazy val scoverageSettings = {
+  import scoverage.ScoverageKeys
+  Seq(
+    // Semicolon-separated list of regexs matching classes to exclude
+    ScoverageKeys.coverageExcludedPackages := """uk\.gov\.hmrc\.BuildInfo;.*\.Routes;.*\.RoutesPrefix;.*Filters?;MicroserviceAuditConnector;Module;GraphiteStartUp;.*\.Reverse[^.]*""",
+    ScoverageKeys.coverageMinimum := 80.00,
+    ScoverageKeys.coverageFailOnMinimum := false,
+    ScoverageKeys.coverageHighlighting := true,
+    parallelExecution in Test := false
+  )
+}
+
+lazy val compileDeps = Seq(
+  ws,
+  "uk.gov.hmrc" %% "bootstrap-play-25" % "1.5.0",
+  "uk.gov.hmrc" %% "govuk-template" % "5.20.0",
```

Comment:
`govuk-template` is on the list of libraries to remove in https://confluence.tools.tax.service.gov.uk/display/PLATOPS/Bootstrap+libraries+migration+guide (step 2)

Reply:
`play-ui` also on the list to remove